File: templates/tests-php/BREACH-002.php
<?php

namespace Compliance\Tests;

use PHPUnit\Framework\TestCase;

class TLSVersionTest extends TestCase
{
    public function testEnforceTLSVersion(): void
    {
        $config = Config::load();
        $tls = $config->authentication->tlsVersion;
        $this->assertContains($tls, ['1.2', '1.3']);
    }
}

File: templates/tests-csharp/BREACH-002.cs
using Xunit;

namespace Compliance.Tests
{
    public class TLSVersionTests
    {
        [Fact]
        public void TestEnforceTLSVersion()
        {
            var config = Config.Load();
            Assert.Contains(config.Authentication.TlsVersion, new[] { "1.2", "1.3" });
        }
    }
}
